specs/latest/2.0/index.html

```diff
@@ -791,13 +885,25 @@ <h2 class="no-toc">Table of contents</h2>
   void drawRangeElements(GLenum mode, GLuint start, GLuint end, GLsizei count, GLenum type, GLintptr offset);
 
   /* Reading back pixels */
-  void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type, GLintptr offset);
+  // WebGL1:
+  void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type,
+                  ArrayBufferView? dstData);
+  // WebGL2:
+  void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type,
+                  GLintptr offset);
+  void readPixels(GLint x, GLint y, GLsizei width, GLsizei height, GLenum format, GLenum type,
+                  ArrayBufferView dstData, GLuint dstOffset);
```

Comment:
Hopefully this overload can be distinguished from the WebGL 1.0 version. If not then maybe we should just add:

void readPixels(GLint x, ..., ArrayBufferView? dstData, optional GLuint dstOffset = 0);